### PR TITLE
Added the responsive images feature

### PR DESCRIPTION
**Pull request by qzminski**
See #1423.

### CODE REVIEW

File: system/modules/isotope/dca/tl_iso_gallery.php [hunk 1]
```diff
@@ -218,7 +218,7 @@
             'label'                 => &$GLOBALS['TL_LANG']['tl_iso_gallery']['zoom_size'],
             'exclude'               => true,
             'inputType'             => 'imageSize',
-            'options'               => $GLOBALS['TL_CROP'],
+            'options'               => \System::getImageSizes(),
```
Comment (Toflar) [inline system/modules/isotope/dca/tl_iso_gallery.php]: Please use the options_callback so this stuff is lazy loaded :) The data is loaded in the FE everytime a Model instance is created and thus relevant for performance :)

**Comment by aschempp** [comment]
Why do we call both `Image::create` and `Picture::create`?

**Comment by qzminski** [comment]
```Image::create``` for BC reasons for custom ```iso_gallery_``` templates. ```Picture::create``` for new gallery templates. I copied this code from contao/core's ```Controller::addImageToTemplate``` method.

**Comment by aschempp** [comment]
You copied? :see_no_evil: 

Why not use the image URL from `Picture::create`?

**Comment by qzminski** [comment]
Apparently there is a reason why ```Controller::addImageToTemplate``` handles it that way.

**Comment by aschempp** [comment]
Removed picture support for inline and elevateZoom gallery in 32d9188b706a1e511507ec223f76a1029770c9a8

**Pull request merged**